WORKSPACE

```diff
@@ -58,7 +58,9 @@ bazel_gazelle()
 
 load("@io_bazel_rules_go//go:deps.bzl", "go_register_toolchains", "go_rules_dependencies")
 go_rules_dependencies()
-go_register_toolchains()
+go_register_toolchains(
+    version = "1.15.7",
```

Comment:
It's now mandatory to specify a version. This is the latest, released today.

Reply:
I've adopted this change whilst updating deps, so that should shorten the diff once that's got CI working :+1: